src/tests/Loader/binding/tracing/BinderTracingTest.ResolutionFlow.cs

```diff
@@ -442,6 +442,7 @@ public static BindOperation AssemblyLoadContextResolvingEvent_CustomALC_Exceptio
         public static BindOperation AssemblyLoadContextResolvingEvent_DefaultALC_Exception()
         {
             var assemblyName = new AssemblyName(SubdirectoryAssemblyName);
+            System.Diagnostics.Debugger.Break();
```

Comment:
If a debugger isn't attached I believe this will trigger an exception. Instead I would branch only if [`Debugger.IsAttached`](https://docs.microsoft.com/dotnet/api/system.diagnostics.debugger.isattached) is `true`.